### ISSUE TITLE
pretrained model on YUD dataset

### ISSUE THREAD

**Issue by ydzhang12345** [fkluger/consac#4]
Hi,

Thank you for your great work! I am curious whether the provided pretrained vanishing point model is trained on NYU or YUD dataset? My use case is mainly on outdoor scene so if possible I would like to use a model trained on YUD. Many thanks!

Another question, I see that in the inference demo we obtain a set of lines, can you give a hint on how to obtain vanishing point from these line or kindly provide a demo on this? Thanks!

**Comment by fkluger** [comment]
As described in the paper, the model is trained on the NYU dataset. Since YUD is pretty small (~100 images with only 25 reserved for parameter tuning) it is not well suited as a training dataset. You could, however, collect a set of outdoor images yourself and use them to train CONSAC in a self-supervised manner. (See Sec. 3.2.2 and Sec. 5.4 in the paper.)  This way you do not need to label any vanishing point ground truth. :-)
